src/components/messagebox/ChattingRoom/index.tsx

```diff
@@ -100,7 +100,11 @@ export const ChattingRoom = ({ roomId, onClose }: ChattingRoomProps) => {
         <IconButton icon="arrowLeft" size={24} onClick={handleCloseRoom} />
         <Styled.Nickname>{user.nickname}</Styled.Nickname>
         <Styled.IconButtonContainer>
-          <IconButton icon="refresh" size={24} onClick={refetchMessage} />
+          <IconButton
+            icon="refresh"
+            size={24}
+            onClick={async () => await getMessageQuery.refetch()}
```

Comment:
```suggestion
onClick={getMessageQuery.refetch}
```
이후에 처리할 상황이 없어서 async, await 제거해도 될 것 같습니다!

Reply:
[반영 완료 ! ](https://github.com/price-offer/offer-fe/pull/196/commits/cd910468f471f328bef8a5281af6182ca20cec16#diff-2e1f52de9ff48c1e322340a310c75b772aa52c734108dbf9888ba96a88db7ce3R106)